app/features/industry/ciip-filing.feature

```diff
@@ -0,0 +1,16 @@
+@mvp
+Feature: CIIP Filing
+
+  As an industry user
+  I want to file a successful CIIP application
+  So that I receive a CleanBC Industrial Incentive Program (CIIP) cheque
+
+  Scenario: know when the CIIP application window will open
+  Scenario: start a CIIP application for a facility that hasn't filed yet
+  Scenario: automatically lock a CIIP application so others can't update it while I am updating it
+  Scenario: resume a CIIP application for a facility that has a filing in progress
+  Scenario: confirm all data reported in my CIIP application
+  Scenario: request certification of data reported in CIIP application
+  Scenario: correct fields an incentive analyst has flagged for review
+  Scenario: receive confirmation that my CIIP application has been filed
+  Scenario: review filed CIIP applications
```

Comment:
Does this include viewing the status of their filed CIIP applications? If so then we need to mark GGIRCS-571 as a duplicate of this scenario in youtrack

Reply:
fixed